### ISSUE TITLE
Temp directories fail to delete zip files created on Macs

### ISSUE THREAD

**Issue by bondjimbond** [ulsdevteam/islandora_datastreams_io#6]
The hidden __MACOSX folder that is automatically added to zip archives on Mac computers cannot be deleted from the tmp directory, despite current efforts to remove it.

It looks like the best way is to avoid extracting it in the first place. Perhaps this might be of some use?

https://stackoverflow.com/questions/37185636/how-to-prevent-extracting-a-php-file-from-zip

**Comment by wgilling** [comment]
Ok, this patch should get rid of those files.  I debated the amount of work that I should do to handle the way that MAC decided to make zip files --- without modifying any logic, I went with the simplest way because it is not my job to work around the way that MAC does zip files.  I let that happen, but just make the delete more absolutely delete the temp folder that was extracted.   34d9d46f234d827b39ed0f4add62e54168dafe52
